src/db/db.js

```diff
@@ -1,15 +1,14 @@
 const { Pool } = require("pg");
-const dbPool = new Pool({
-  connectionString: process.env.DB_CONNECTION_STRING,
-});
+const dbPool = new Pool();
 
 // this could be a heartbeat/health-check endpoint?
 dbPool.connect((err, client, release) => {
```

Comment:
this is a rough fix for a larger problem we'll want to handle around error states.